river/multioutput/chain.py

```diff
@@ -231,8 +252,199 @@ def predict_one(self, x):
         x = copy.copy(x)
         y_pred = {}
 
+        if not isinstance(self.order, list):
+            return y_pred
+
         for o, clf in self.items():
             y_pred[o] = clf.predict_one(x)
             x[o] = y_pred[o]
 
         return y_pred
+
+
+class ProbabilisticClassifierChain(ClassifierChain):
+    r"""Probabilistic Classifier Chains.
+
+    The Probabilistic Classifier Chains (PCC) [1^] is a Bayes-optimal method
+    based on the Classifier Chains (CC).
+
+    Consider the concept of chaining classifiers as searching a path in a
+    binary tree whose leaf nodes are associated with a label $y \in Y$. While
+    CC searches only a single path in the aforementioned binary tree, PCC looks
+    at each of the $2^l$ paths, where $l$ is the number of labels. This limits
+    the applicability of the method to data sets with a small to moderate
+    number of labels. The authors recommend no more than about 15 labels for
+    real-world applications.
+
+    Parameters
+    ----------
+    model
+
+    Examples
+    --------
+    >>> from river import feature_selection
+    >>> from river import linear_model
+    >>> from river import metrics
+    >>> from river import multioutput
+    >>> from river import preprocessing
+    >>> from river import synth
+
+    >>> dataset = synth.Logical(seed=42, n_tiles=100)
+
+    >>> model = multioutput.ProbabilisticClassifierChain(
+    ...     model=linear_model.LogisticRegression()
+    ... )
+
+    >>> metric = metrics.Jaccard()
+
+    >>> for x, y in dataset:
+    ...    # Convert y values to booleans
```

Comment:
Is this comment valid?

Reply:
No, it was unintentionally left. I removed it.